src/migration/ember-engine/steps/create-options.js

```diff
@@ -1,11 +1,19 @@
-import { readPackageJson } from '@codemod-utils/json';
+import { readPackageJson, validatePackageJson } from '@codemod-utils/json';
+
+function analyzePackageJson(codemodOptions) {
+  const packageJson = readPackageJson(codemodOptions);
+
+  validatePackageJson(packageJson);
+
+  return packageJson.name;
+}
```

Comment:
**Technical risk**

Validation is needed for `ember-engine`, but not for `ember-addon` and `ember-app`. The reason is, currently, only `ember-engine` takes care of updating the import statements after un-podding a project.

Consider either,

- Remove `updateAbsolutePaths` and `useRelativePaths` from `ember-engine`
- Implement `updateAbsolutePaths` and `useRelativePaths` for `ember-addon` and `ember-app`